absl/debugging/internal/stacktrace_powerpc-inl.inc

```diff
@@ -150,8 +150,8 @@ static void **NextStackFrame(void **old_sp, const void *uc) {
 }
 
 // This ensures that absl::GetStackTrace sets up the Link Register properly.
-void StacktracePowerPCDummyFunction() __attribute__((noinline));
-void StacktracePowerPCDummyFunction() { __asm__ volatile(""); }
+void AbslStacktracePowerPCDummyFunction() __attribute__((noinline));
```

Comment:
Thank you very much for finding this!

I suspect we can do even better by marking this static and using the portable symbols we've added since this code was written. Something like:
```
ABSL_ATTRIBUTE_NOINLINE static void AbslStacktracePowerPCDummyFunction() {
  ABSL_BLOCK_TAIL_CALL_OPTIMIZATION();
}
```

I'll run this though the global presubmit tonight since we don't have ppc64le test coverage for OSS contributions.

Reply:
Sure. Let me know if there's anything else I can do!